Zend/Optimizer/zend_dump.c

```diff
@@ -410,6 +410,10 @@ ZEND_API void zend_dump_op(const zend_op_array *op_array, const zend_basic_block
 	uint32_t flags = zend_get_opcode_flags(opline->opcode);
 	uint32_t n = 0;
 
+	if (dump_flags & ZEND_DUMP_LINE_NUMBERS) {
+		fprintf(stderr, "L%04u ", opline->lineno);
+	}
```

Comment:
Shouldn't be needed anymore.

Reply:
Fixed in 6a3898195f29dd11b98f3a3f46d10c5ec732f881